README.md

```diff
@@ -1,27 +1,37 @@
 # Contenta
 
-## Quick Install
+## Requirements
 
 - Install [composer](https://getcomposer.org/)
 - Make sure you have the sqlite extension for PHP (if you're using the default install on a mac, this should already be there)
```

Comment:
As a follow up, let's move this requirement to the quick install ... the normal installation doesn't actually require sqlite, anyway ...